convertor/lib/models/price_reduction.dart

```diff
@@ -0,0 +1,22 @@
+class PriceReduction{
+
+    double price; // prix de départ
+    int reduction; // valeur de la réduction
+
+    PriceReduction(double myPrice, int myReduction){
+        
+      this.price = myPrice;
+      this.reduction = myReduction;
+    }
+
+    Map<String, Object> Calcul(double myPrice, int myReduction){
```

Comment:
```suggestion
    Map<String, Object> calcul(double myPrice, int myReduction){
```
Pas de majuscule à la première lettre d'une fonction